src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img-container">
+        <img
+          class="card__img"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        >
+      </div>
+
+      <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__reviews">
+        <!-- STARS -->
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <!-- STARS -->
+        <div class="card_rewies_info">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-value">$2,199</div>
+      </div>
+
+      <a class="card__buy" data-qa="hover">
+        <div class="card__buy-text">BUY</div>
+      </a>
```

Comment:
do not place block elements inside inline elements. div is redudant here
```suggestion
      <a class="card__buy" data-qa="hover">
        buy
      </a>
```